### ISSUE TITLE
Folder structure

### ISSUE THREAD

**Issue by Walidmash** [facg3/todo-list#13]
Your .html .css .js files are all in the root directory, you should organize these files into separated folders.
even if this project is small, you need to learn and get used to an organized and scalable folder structure schema.

**Comment by MMortaga** [comment]
@Walidmash Correct. However, for this project we were supposed to clone the "skeleton" folder and just edit the codes inside. We were not expected to separate the files or alter the default structure.
